draft-ietf-rtcweb-jsep.xml

```diff
@@ -5886,6 +5886,12 @@ a=msid:-
 
           <t>If the packet is of type APP, deliver a copy of the packet
           to each "m=" line.</t>
+
+          <t> Some multiparty RTP topologies result in endoints
+          receiving RTCP reports for an SSRC that is not associated
+          with the endpoint that received the RTCP packet. These
+          packets MAY be discarded. </t>
```

Comment:
If an RTCP packet within a compound packet matches one or more RtpSender/RtpReceivers based on the routing algorithm, the compound packet will be delivered.  If there are no matches, it will be discarded.  So not sure that the "MAY" makes sense (aren't the processing rules mandatory-to-implement?) or how this addition changes the algorithm.